mollie-payments-for-woocommerce.php

```diff
@@ -58,6 +58,32 @@ function mollie_wc_plugin_activation_hook()
     deleteWPTranslationFiles();
 }
 
+function showWcDroppingNotice()
+{
+    if (get_option('mollie-payments-for-woocommerce-wc-drop')) {
+        return false;
```

Comment:
Probably, just `return;` should be enough. As I can see, `false` is returned in all cases and the function return value is never used. It's not a big problem, though. It shouldn't break anything, just a very small, but unnecessary part of code.